### PR TITLE
Implement base styles for material theme in dxScheduler

### PR DESCRIPTION
**Pull request by babich-a**

### CODE REVIEW

File: js/ui/scheduler/ui.scheduler.header.js [hunk 1]
```diff
@@ -210,6 +212,7 @@ var SchedulerHeader = Widget.inherit({
 
         this._viewSwitcher = this._createComponent($element, DropDownMenu, {
             onItemClick: this._updateCurrentView.bind(this),
+            buttonIcon: themes.isMaterial() ? "chevrondown" : "overflow",
```
Comment (Yarovaya) [inline js/ui/scheduler/ui.scheduler.header.js]: Can you make themes.isMaterial() checking in a default options config? And then use some option for such checking?

File: js/ui/scheduler/ui.scheduler.appointment_tooltip.js [hunk 2]
```diff
@@ -100,9 +106,18 @@ var appointmentTooltip = {
...
 
-        var $buttons = $("<div>")
-                        .addClass(APPOINTMENT_TOOLTIP_BUTTONS_CLASS)
-                        .appendTo($tooltip);
+        var $buttons = $("<div>").addClass(APPOINTMENT_TOOLTIP_BUTTONS_CLASS);
+
+        if(themes.isMaterial()) {
+            $buttons.prependTo($tooltip);
+            this._getCloseButton().appendTo($buttons);
+            this._getTooltipBackgroundColor($appointment).done(function(color) {
```
Comment (Yarovaya) [inline js/ui/scheduler/ui.scheduler.appointment_tooltip.js]: Try to simplify it using directly observer menthod getAppointmentColor calling.

File: js/ui/scheduler/ui.scheduler.appointment_tooltip.js [hunk 3]
```diff
@@ -48,7 +51,8 @@ var appointmentTooltip = {
                 at: "top",
                 of: $appointment,
                 boundary: isAllDay ? instance.$element() : instance.getWorkSpaceScrollableContainer(),
-                collision: "fit flipfit"
+                collision: "fit flipfit",
+                offset: { x: 0, y: this.instance.option("_appointmentTooltipVerticalOffset") }
```
Comment (lukyanovas) [inline js/ui/scheduler/ui.scheduler.appointment_tooltip.js]: I think better to use  _appointmentTooltipOffset and set it to { x: 0, y: 11 } for material

File: js/ui/scheduler/ui.scheduler.header.js [hunk 4]
```diff
@@ -173,7 +173,8 @@ var SchedulerHeader = Widget.inherit({
             },
             selectedItem: this.option("currentView"),
             tabIndex: this.option("tabIndex"),
-            focusStateEnabled: this.option("focusStateEnabled")
+            focusStateEnabled: this.option("focusStateEnabled"),
+            showNavButtons: true
```
Comment (lukyanovas) [inline js/ui/scheduler/ui.scheduler.header.js:177]: It is the very strange change

File: js/ui/scheduler/ui.scheduler.appointment_tooltip.js (hunk 3 above)
Comment (babich-a) [reply js/ui/scheduler/ui.scheduler.appointment_tooltip.js]: fixed

File: js/ui/scheduler/ui.scheduler.header.js (hunk 4 above)
Comment (babich-a) [reply js/ui/scheduler/ui.scheduler.header.js:177]: Why? We need navigation buttons, but showNavButtons is 'false' by default for some themes (material)

Comment (lukyanovas) [reply js/ui/scheduler/ui.scheduler.header.js:177]: I think set the option to true here can change behavior on mobile devices

Comment (babich-a) [reply js/ui/scheduler/ui.scheduler.header.js:177]: I agree. I'll remove this after we select the right default combination of scrollByContent and showNavButtons options for dxTabs in material theme.

**Review by KsyushaM — APPROVED** [review]

**Pull request merged**